test-workspace/scripts/randomNote.js

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env node
+/** Creates a randomly generated note.
+ *
+ * The generated note will contain paragraphs, lists, tables, wikilinks, block
+ * anchors etc., trying to make use of Dendron features.
+ * 
+ * There is currently no options for this script, but you can edit the following variables to edit what gets generated.
+ */
+/* eslint-disable no-console */
+/* eslint-disable no-lone-blocks */
+/* eslint-disable no-plusplus */
+
+/** How many markdown elements to generate. */
```

Comment:
could you add some docs in `[[dendron.dev.qa.perf]]` on using this? (doesn't exist yet)